src/main/java/io/nats/client/Options.java

```diff
@@ -2117,6 +2136,14 @@ public boolean isTlsFirst() {
         return tlsFirst;
     }
 
+    /**
+     * Get whether to throw {@link java.util.concurrent.TimeoutException} on timeout instead of {@link java.util.concurrent.CancellationException}.
+     * @return the flag
+     */
+    public boolean isUseTimeoutException() {

```

Comment:
To be more consistent with other booleans in this class, please use `useTimeoutException` no `is`